doc-content/enterprise-only/admin-and-config/kie-server-deactivate-kie-containers-proc.adoc

```diff
@@ -13,4 +13,4 @@ A container has been created and configured in {CENTRAL}.
 . From the *Deployment Units* pane, select the deployment unit you want to activate or deactivate.
 . Click *Activate* or *Deactivate* in the upper-right corner of the deployment unit pane.
 +
-Once you deactivate a given KIE container, you cannot create new process instances from it.
+Once you deactivate a given KIE container, you cannot create process instances from it.
```

Comment:
You cannot create a process instance once you deactivate a KIE container.

Reply:
How about "You cannot create a process instance from a KIE container once it is deactivated."?